documents/operating.md

```diff
@@ -29,15 +29,67 @@ SECTION 2: 🤝🤝🤝 Partnership 🤝🤝🤝
 ---------------------------------------------------------------------
 SECTION 3: 🤔🤔🤔 Projects 🤔🤔🤔
 ---------------------------------------------------------------------
-- [ ] Types of project
-- [ ] Evaluating ideas
-- [ ] IP (Intellectual Property / Internet Protocol)
-- [ ] Open source
+- [x] Types of project
+- [x] Evaluating ideas
+- [x] IP (Intellectual Property / Internet Protocol)
+- [x] Open source
 - [ ] When a project should be considered retired
 - [ ] Removing a project from BIFFUD
 - [ ] Adopting IP from existing entities / projects
 - [ ] Reallocating copyright / IP upon spinoff
 
+### Types of Project
+A. **Grant Funded:** projects funded by grants from third parties are executed
+by the BIFFUD member who has led the relationship with the funders, or by an
+alternative BIFFUD member as delegated by that applicant.  Money associated with
+a grant is to be distributed as per the grant's agreement.
+
+B. **Member Owned:** projects that are led or adopted by BIFFUD members.  The
+owners of a project must all be BIFFUD members, and the percentages and specific
+terms of ownership should be define by those members as part of their application
+for the adoption of the idea under BIFFUD.
+
+C. **BIFFUD Core:** projects that are taken on by the entire corporation.  These
+might be projects utilized by the Corporate Overlords to execute on company policy,
+or simply be projects that the Corporate Overlords decide to develop without a concept
+of individual ownership.
+
+
+### Evaluating Ideas
+Anybody can build anything they want at any time, but in order for something to become
+an official BIFFUD project it must be formally adopted by the Corporate Overlords during
+a plotting session.  The determination of what makes a project a BIFFUD project is
+ultimately determined by that process, but in the spirit of framing the conversation,
+the following questions are recommended.
+
+1. How is this project a bad idea?
+2. Does it make you thinking face emoji?
+3. Is it realistic to implement through BIFFUD?
+4. If the project were a character in Dungeons and Dragons, what would it's alignment be?
+
+The BIFFUD brand deserves compelling answers to questions 1 and 2. Projects should be
+realistic to implement, and should either be "Chaotic Good", "Neutral Good", or
+"Chaotic Neutral".  All of these data points simply serve as guidance. The ultimate
+decision of what projects should be adopted falls on the Corporate Overlords.
+
```

Comment:
What is the formal framework for adoption? Vote? Maybe this is being addressed in later sections.